src/Controller/Async/General.php

```diff
@@ -393,9 +393,11 @@ private function fetchNews($hostname)
             ];
         }
 
-        $fetchedNewsItems = json_decode($fetchedNewsData);
-        if ($newsVariable = $this->getOption('general/branding/news_variable')) {
-            $fetchedNewsItems = $fetchedNewsItems->$newsVariable;
+        $fetchedNewsItems = json_decode($fetchedNewsData, true);
```

Comment:
Oh dear … it seems I missed this one in the `bolt/common` cutover … or I derped a merge :disappointed: 

`$fetchedNewsItems = \Bolt\Common\Json::parse($fetchedNewsData)` will do the same job, plus it will throw a catchable exception if there is _any_ problem in the decoding … at which point (the catch) you can return something friendly to the end user.

Reply:
Ah, derp! I knew that.. 

I'll work on this PR some more.

Reply:
Oh, it is totally not your fault here … mine if anyone's.

Reply:
Regardless, it's a good pointer to improve the current PR, so i'll work on it. ;-)